src/com/theironyard/novauc/Logic.java

```diff
@@ -0,0 +1,106 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by dangelojoyce on 2/23/17.
+ */
+
+
+public class Logic {
+
+    public boolean frontAgain(String str) { //THE D'ANGELO METHOD
+
+        String strStart;
+//        if (str.length() > 0) {
+//            strStart = str.substring(0, 2).toLowerCase();
+//            System.out.println("Your string ends with: " + str.substring((str.length() -2), str.length()));
+//            System.out.println("Your string starts with: "+ strStart);
+//        }
+        if (str.endsWith(str.substring(0, 2).toLowerCase())) {
+            return true;
+        } else if (str.endsWith(str.substring(0).toLowerCase()))
+            return true;
+
+
+        //System.out.println("Here is your " +substring)
+        return false;
+    }
+
```

Comment:
D'Angelo this was an open pull request but there is no submission to newline.  You need to solve 5 more logic problems and write tests for all of the problems.  Also please copy and paste the problem you are trying to solve above each method and comment it out that way I know what the method is supposed to do